org.lflang/src/org/lflang/FileConfig.java

```diff
@@ -374,6 +397,45 @@ public static void createDirectories(Path path) {
             file.mkdirs();
         }
     }
+    
+    /**
+     * Copy a given directory from 'src' to 'dest'.
+     * 
+     * @param src The source directory path.
+     * @param dest The destination directory path.
+     * @throws IOException if copy fails.
+     */
+    public static void copyDirectory(Path src, Path dest) throws IOException {
+        try (Stream<Path> stream = Files.walk(src)) {
+            stream.forEach(source -> copyFile(source, dest.resolve(src.relativize(source))));
+        }
+    }
+
+    /**
+     * Copy a given file from 'src' to 'dest'.
+     * 
+     * @param source The source file path string.
+     * @param dest The destination file path string.
+     * @throws RuntimeException if copy fails.
+     */
+    public static void copyFile(String src, String dest) {
+        copyFile(Paths.get(src), Paths.get(dest));
+    }
+
+    /**
+     * Copy a given file from 'src' to 'dest'.
+     * 
+     * @param source The source file path.
+     * @param dest The destination file path.
+     * @throws RuntimeException if copy fails.
+     */
+    public static void copyFile(Path src, Path dest) {
+        try {
+            Files.copy(src, dest, java.nio.file.StandardCopyOption.REPLACE_EXISTING);
+        } catch (Exception e) {
+            throw new RuntimeException(e.getMessage(), e);
```

Comment:
Why is RuntimeException thrown here and not IOException as in copyDirectory()? Also catching all types of exceptions is a code smell, as it has the potential to mask problems.

Reply:
This was to avoid checked [exceptions](https://www.baeldung.com/java-lambda-exceptions#handling-checked-exceptions) in a lambda expression. I have pushed a fix.